packages/api-request-builder/test/create-service.spec.js

```diff
@@ -131,6 +135,27 @@ describe('createService', () => {
         )
       })
 
+      // query-location
+      it('should support `country`', () => {
```

Comment:
These tests should fail. Haven't tested it on my machine, but you didn't add `queryLocation` to the `features` on L98

Reply:
strange - for me it was working .. but I added it here https://github.com/commercetools/nodejs/pull/381/commits/d86d95a6e56544a34f4251ebc8b019391699cc57#diff-fce1f76e56627dde45f04ff65082bc13R102

Reply:
It's related to [this comment](https://github.com/commercetools/nodejs/pull/381#discussion_r156667277)
But nice that you added it